src/core/lib/iomgr/tcp_windows.cc

```diff
@@ -112,7 +113,10 @@ typedef struct grpc_tcp {
 
   grpc_closure* read_cb;
   grpc_closure* write_cb;
-  grpc_slice read_slice;
+
+  /* garbage after the last read */
```

Comment:
Not sure why it's called "garbage". But I'm okay with this since it's used elsewhere before this PR.